docs/specs/3-13-2023-commands-fit-n-finsh.md

```diff
@@ -0,0 +1,71 @@
+# 3/13/2023 - Commands Fit-and-Finish Proposal
+
+## Design
+
+Commands that interact with the core development workflow will be verb-based. This feels natural to the user as the user is building an application, provisioning infrastructure, and deploying their application to Azure -- which are all actions.
+
+For commands that are of configuration and management in-nature, it is fine to add a noun for a subcommand. For example, `azd env <action>` and `azd config <action>`.
+
+This means that the overall proposed design matches with what `azd` currently offers, and no changes need to occur.
+
+## `azd` usage output
+
+To help introduce user to the command layout, I propose slight changes to the current `azd` usage output (with UX input needed), that looks like the following:
+
+```
+Commands
+  Develop an app
+    init     	: Initialize a new application.
+    provision	: Provision the Azure resources for an application.
+    deploy   	: Deploy the application's code to Azure.
+    up       	: Initialize application, provision Azure resources, and deploy your project with a single command.
+
+  Monitor, test, and release your app
+    monitor  	: Monitor a deployed application.
+    pipeline 	: Manage and configure your deployment pipelines.
+
+  Authenticate with Azure
+    login    	: Log in to Azure.
```

Comment:
We have the hidden `auth show` command.  Perhaps we should move these into (the currently hidden) `auth` command?  It would make the login command a little worse (`azd auth login` vs `azd login`) but this does match `gh`.  

@savannahostrowski had thought it would be nice if the tool just auto logged you in when it needed to do so (we'd have to pick a reasonable default flow, like browser based) and in that case `azd auth login` being a bit more typing isn't the end of the world.  Interested in your thoughts there.

Reply:
another option is azd auth with no args is login. Just a thought.

Reply:
If its just one command, I wouldn't have a whole section for it.